code/edgeidentity/src/androidTest/java/com/adobe/marketing/mobile/edge/identity/IdentityAndroidTestUtil.java

```diff
@@ -0,0 +1,303 @@
+/*
+  Copyright 2021 Adobe. All rights reserved.
```

Comment:
Year `2022`*

Reply:
Thanks for catching this! This file shouldn't have been committed at all. I removed this in the new revision.